partials/content.hbs

```diff
@@ -42,6 +42,18 @@
                 {{custom_excerpt}}
             </div>
         {{/if}}
+        {{#if @custom.show_author}}
+            <div class="single-footer-top">
+                <h3 class="single-footer-title">Published by:</h3>
+                <div class="author-list">
+                    {{#foreach authors}}
+                            <a href="{{url}}" title="{{name}}">
+                                <p>{{name}}</p>
+                            </a>
```

Comment:
Some extra indentation here, not sure if you're using tabs? In any case, do reduce this down to 4 space indentations.